### PR TITLE
Remove StarkWare dependencies

### PR DESCRIPTION
**Pull request by teghnet**
- Remove StarkWare dependencies

### CODE REVIEW

File: omnia/config/feed.conf [hunk 1]
```diff
@@ -25,7 +25,6 @@
     "LINK/USD":{"msgExpiration":1800,"msgSpread":0.5},
     "MANA/USD":{"msgExpiration":1800,"msgSpread":0.5},
     "MATIC/USD":{"msgExpiration":1800,"msgSpread":0.5},
-    "UNI/USD":{"msgExpiration":1800,"msgSpread":0.5},
```
Comment (madumas) [inline omnia/config/feed.conf:28]: seems unrelated to starkware. is that a mistake?

Comment (teghnet) [reply omnia/config/feed.conf]: I wanted to clean up, but reverted that change as it is not related to starkware

**Review by MDobak — APPROVED** [review]
